src/server/storage/memory.js

```diff
@@ -100,6 +101,69 @@ function Memory(mainLogger, gmeConfig) {
             return deferred.promise.nodeify(callback);
         };
 
+        this.loadPaths = function (rootHash, paths, excludes, callback) {

```

Comment:
My first intent was to put this function completely into the storage, but it turned out that if I want it to function for all types of users (bin scripts, tests, worker, webclient) there is no single point where the function could be implemented.
If we fine with the pre-loading not working for every type of users (mostly the webclient could gain from it), then I can put the functionality into the storage and have an empty function for the others.